webapp/main.go

```diff
@@ -14,7 +14,8 @@ var templates = template.Must(template.ParseGlob("templates/*.html"))
 func init() {
 	// Test run results, viewed by browser (default view)
 	// For run results diff view, 'before' and 'after' params can be given.
-	http.HandleFunc("/", testHandler)
+	http.HandleFunc("/", testResultsHandler)
```

Comment:
Would it make more sense to just have two handlers, if `strings.Index(r.URL.Path, "/results/") != 0` can only match the first case with an early return?

Reply:
I don't think it matters much. More boilerplate in a function + docs than in an if-statement.

Reply:
OK :)